test/plugin/test_driver.rb

```diff
@@ -16,6 +16,19 @@
 require 'fluent/event'
 require 'fluent/test/input_test'
 
+module Fluent
+  module Test
+    # rubocop:disable Style/ClassVars
+    class BufferedOutputTestDriver < InputTestDriver
+      @@run_method = BufferedOutputTestDriver.instance_method(:run)
```

Comment:
`@run_method` is considered class instance variable, while `@@run_method` is considered class variable.
eg.: http://www.railstips.org/blog/archives/2006/11/18/class-and-instance-variables-in-ruby/

For our case, we do want to use a class variable in order to call the original method (the method belongs to the class) later on, disabling the rubocop complaints sounds like a reasonable approach to me.

Reply:
Thanks for confirming that for me!